boilerplate/openshift/golang_osd_cluster_operator/standard.mk

```diff
@@ -71,8 +71,7 @@ docker-push: push
 
 .PHONY: gocheck
 gocheck: ## Lint code
-	boilerplate/openshift/golang_osd_cluster_operator/golint.sh
-	${GOENV} go vet ./cmd/... ./pkg/...
```

Comment:
So we're counting on golangci-lint being a superset of go vet, yah?

Reply:
Yep, it's a [superset](https://golangci-lint.run/usage/linters#enabled-by-default-linters)